apps/backend/src/modules/activities/activity.repository.ts

```diff
@@ -242,6 +247,78 @@ class ActivityRepository implements Repository<ActivityEntity> {
 		});
 	}
 
+	public async findAllWithPagination({
```

Comment:
```suggestion
	public async findAllWith({
```
should be with params. it's by default with pagination